third_party/docfx/templates/devsite/ManagedReference.html.primary.tmpl

```diff
@@ -1,7 +1,8 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 {{!master(layout/_master.tmpl)}}
 
-<h1 class="page-title">{{#title}}{{title}}{{/title}}{{^title}}{{>partials/title}}{{/title}}</h1>
+<h1 class="page-title">{{title}}
+</h1>

```

Comment:
Is the `</h1>` closing in a separate line needed?

Reply:
In order to match our current generated HTML, yes. We can put it on a separate line as well, this would then be updated for all our documentation as well.

Reply:
The "expected" at the moment is on a separate line. Let's fix this.